tests/reg_tests/reg_test_utils.py

```diff
@@ -13,6 +13,24 @@ def assert_adjoint_sens_allclose(handler, CFDSolver, ap, evalFuncs=None, **kwarg
     handler.root_print("Eval Functions Sens:")
     handler.root_add_dict("Eval Functions Sens:", funcsSens, rtol=rtol, atol=atol)
 
+def assert_adjoint2_sens_allclose(handler, CFDSolver, ap, evalFuncs=None, **kwargs):
+    rtol, atol = getTol(**kwargs)
+    funcsSens = {}
+    CFDSolver.evalFunctionsSens(ap, funcsSens, evalFuncs=None)
+    CFDSolver.evalFunctionsSens(ap, funcsSens, evalFuncs=None)
+    handler.root_print("Eval Functions Sens 2:")
+    handler.root_add_dict("Eval Functions Sens 2:", funcsSens, rtol=rtol, atol=atol)
```

Comment:
Shouldn't we compare the second run against the values of the first run `Eval Functions Sens`? I don't see the point of adding this if the point of the test is to make sure that the adjoint converges to the same result (within a tolerance)